### PR TITLE
Code coverage: CardTransactionPreview Component - 100%

### PR DESCRIPTION
**Pull request by Dimple16**
Screenshot 2023-02-27 at 10 59 22 PM

### CODE REVIEW

File: src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts [hunk 1]
```diff
@@ -1,26 +1,61 @@
...
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should render transaction details', () => {
+    const mockTransactionDetails: Partial<MatchedCCCTransaction> = {
```
Comment (rvab) [inline src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts:31]: Don't we have a mock object for this in the services?

File: src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts [hunk 2]
```diff
@@ -1,26 +1,61 @@
...
+    expect(getTextContent(vendor)).toBe('Amazon,');
+
+    const date = getElementBySelector(fixture, '.card-transaction--matching-content span:nth-child(2)');
+    expect(getTextContent(date)).toBe('Feb 20, 2022 -');
+
+    const amount = getElementBySelector(fixture, '.card-transaction--matching-content span:nth-child(3)');
+    expect(getTextContent(amount)).toBe('$100.00');
+  });
+
+  it('should not render transaction details when transactionDetails is falsy', () => {
```
Comment (rvab) [inline src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts:54]: `when transactionDetails is null`

**Review by rvab — APPROVED** [review]
minor comment

File: src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/fyle/merge-expense/card-transaction-preview/card-transaction-preview.component.spec.ts:31]: no, just used it in one place

**Pull request merged**